tests/test_toggle.py

```diff
@@ -275,6 +275,28 @@ def test_multi_level(self):
                           'level1p.level2p.test_int32',
                           'level1p.test_bool', 'test_int8'])
 
+    def test_get_value(self):
+        tg = modmesh.Toggle.instance.clone()
+        tg.dynamic_clear()
+
+        tg.add_subkey("level1")
+        tg.set_int8("level1.test_int8", 21)
+        self.assertEqual(tg.level1.test_int8, 21)
+        self.assertEqual(tg.get_value('level1.test_int8'), 21)
+        self.assertEqual(tg.get_value(key='level1.test_int8'), 21)
+        self.assertEqual(tg.get_value('level1.test_int8', 22), 21)
```

Comment:
Add tests for the default value.  If the value is there, the default value is not used.